Library/Homebrew/cask/lib/hbc/artifact/binary.rb

```diff
@@ -9,7 +9,7 @@ def install_phase
 
       def link
         super
-        FileUtils.chmod "+x", source
+        @command.run!("/bin/chmod", args: ["+x", source])
```

Comment:
To skip it, simply return out of the method:

```ruby
return if source.executable?
```

And `SystemCommand.run` should still be `@command.run!`, other than that this looks good.